Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/BaseLinearElasticityFEMForceField.inl

```diff
@@ -82,10 +132,68 @@ auto BaseLinearElasticityFEMForceField<DataTypes>::getYoungModulusInElement(sofa
     }
     else
     {
-        setYoungModulus(5000);
+        setYoungModulus(defaultYoungModulusValue);
```

Comment:
A get at indice X that end to do a set at indice 0 is so surprising.

Reply:
yes, and it prevents the method to be const...

Reply:
Actually, I can simply return `defaultYoungModulusValue` and not changing the vector. What do you think?